### ISSUE TITLE
Ability to disable multiple roles by default

### ISSUE THREAD

**Issue by rizaardiyanto1412** [publishpress/PublishPress-Capabilities#622]
> I searched for a while for a way to disable multiple roles, before realizing that it’s something included in default.
> 
> https://publishpress.com/knowledge-base/adding-multiple-roles-to-a-user/
> 
> this option enabled by default is not desired in my scenario
> it creates confusion for my “editor” role client which I only planned to give him the power to create other “editor” or “author” roles only.
> 
> I would like to have the possibility and the flexibility to activate or deactivate it specifically for specific roles and according to certain scenario.

Original thread: https://wordpress.org/support/topic/ability-to-disable-multiple-roles-by-default/

**Comment by olatechpro** [comment]
@rizaardiyanto1412 I think we already implemented this.

Can you check role tabs in capabilities settings and confirm if that's what the request is about?

**Comment by rizaardiyanto1412** [comment]
@olatechpro The role tabs in the capabilities setting is to give an option to user for enable multiple capabilities in Add new user (wp-admin/user-new.php)

While this request is about the option to deactivate multiple role capabilities completely (just like when a site does not have PublishPress Capabilities).

**Comment by olatechpro** [comment]
@rizaardiyanto1412 When you uncheck this box, are you still able to add multiple roles to WordPress user when adding/editing user?

If not, maybe you should explain what you mean by

> option to deactivate multiple role capabilities completely (just like when a site does not have PublishPress Capabilities).

as I'm trying to understand the capabilities plugin impact or effect even with the feature disabled

**Comment by rizaardiyanto1412** [comment]
@olatechpro Yes, even with box unchecked, I can still have multiple role. Check this video: https://www.loom.com/share/bd4c3c789f9e4a03a13cbfab72456a51

**Comment by olatechpro** [comment]
Thanks @rizaardiyanto1412  I think i see the issue now, the settings only control when new user is been added and not when editing user.

@stevejburge Do we need to modify this settings description and extend it feature to including when editing user or we should add new control settings?

![mr](https://user-images.githubusercontent.com/46832636/213676234-43cb9a02-072c-4b40-9d4c-d31efdd54b1b.jpg)

Also note, once this feature is implemented, editing a user with multiple role will only show the first role and updating the settings will remove other roles.

**Comment by stevejburge** [comment]
@olatechpro Yes, we should fix bug and change the description. I don't think a new settings is needed.
